--- klamav-0.09/src/freshklam.cpp~	2005-01-01 20:00:12.000000000 +0200
+++ klamav-0.09/src/freshklam.cpp	2005-01-01 20:00:14.000000000 +0200
@@ -88,7 +88,7 @@
 	lastDownloadPaths = config->readListEntry("lastDownloadPaths");
 
 	if (lastDownloadPaths.isEmpty()){
-		lastDownloadPaths.append( "/usr/local/share/clamav");
+		lastDownloadPaths.append( "/var/lib/clamav");
 		config->writeEntry("lastDownloadPaths", lastDownloadPaths);
 		lastDownloadPaths = config->readListEntry("lastDownloadPaths");
 	}
--- klamav-0.09/src/klammail-0.06/src/clamdmail.c~	2004-12-29 22:01:35.000000000 +0200
+++ klamav-0.09/src/klammail-0.06/src/clamdmail.c	2005-01-01 20:04:03.000000000 +0200
@@ -548,7 +548,7 @@
 			break;
 			}
 	}else{
-		fprintf(tmp,"DatabaseDirectory /usr/local/share/clamav\n");
+		fprintf(tmp,"DatabaseDirectory /var/lib/clamav\n");
 	}
 	fprintf(tmp,"ScanMail\n");
 	fprintf(tmp,"FixStaleSocket\n");
